src/oceans/init.js

```diff
@@ -7,6 +7,7 @@ import {setInitialState, setSetStateCallback} from './state';
 import {render as renderCanvas} from './renderer';
 import modeHelpers from './modeHelpers';
 import soundLibrary from './models/soundLibrary';
+import * as I18n from './i18n';
```

Comment:
any reason not to just build this as a class if we're going to treat it as a class?

Reply:
Changed this a bit to be consistent with most of our other files (like `soundLibrary`).